### PR TITLE
chore: windows dev setup support

### PR DESCRIPTION
**Pull request by yuth**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ammarkarachi** [comment]
lerna ERR! yarn run build exited 1 in 'amplify-appsync-simulator'
lerna ERR! yarn run build stdout:
yarn run v1.19.1
$ yarn copy-assets && tsc
$ rm -rf ./public && cp -R ../amplify-graphiql-explorer/build ./public
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:
'rm' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in 'amplify-appsync-simulator'
lerna WARN complete Waiting for 2 child processes to exit. CTRL-C to exit immediately.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! amplify-cli@0.1.0 setup-dev-win: `lerna exec -- del /f package-lock.json && lerna bootst
ify-cli && del /f package-lock.json && npm link && cd ../.. && lerna run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the amplify-cli@0.1.0 setup-dev-win script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above

npm ERR! A complete log of this run can be found in:
npm ERR!     D:\Users\ammkara\AppData\Roaming\npm-cache\_logs\2019-11-05T23_23_56_346Z-debug.log

**Comment by ammarkarachi** [comment]
It worked I was on the wrong branch,

File: package.json [hunk 1]
```diff
@@ -60,7 +61,10 @@
     "eslint-plugin-prettier": "^3.1.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.0.1",
-    "pretty-quick": "^1.11.1"
+    "pretty-quick": "^1.11.1",
```
Comment (attilah) [inline package.json]: Please review these deps.

File: package.json [hunk 2]
```diff
@@ -14,8 +14,9 @@
     "dev-build": "yarn && lerna run build",
     "link-dev": "cd packages/amplify-cli && ln -s $(pwd)/bin/amplify $(yarn global bin)/amplify-dev && cd -",
     "rm-dev-link": "rm -f $(yarn global bin)/amplify-dev",
```
Comment (attilah) [inline package.json]: I think imraf needed here as well?

File: package.json [hunk 3]
```diff
@@ -14,8 +14,9 @@
     "dev-build": "yarn && lerna run build",
     "link-dev": "cd packages/amplify-cli && ln -s $(pwd)/bin/amplify $(yarn global bin)/amplify-dev && cd -",
```
Comment (attilah) [inline package.json]: does 'ln' exist on windows?

File: scripts/link-bin.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  console.log(`${src} does not exits`);
+  process.exit(1);
+}
+
+if (!fs.existsSync(path.dirname(dest))) {
+  fs.mkdirpSync(path.dirname(dest));
+}
+
+if (process.platform === 'win32') {
+  cmdShim(src, dest, err => {
```
Comment (attilah) [inline scripts/link-bin.js]: We've to make sure this works from all the possible other shells: PowerShell and WSL

File: yarn.lock [hunk 5]
```diff
@@ -5874,6 +5874,18 @@ copy-to-clipboard@^3.0.8, copy-to-clipboard@^3.2.0:
   dependencies:
     toggle-selection "^1.0.6"
 
+copyfiles@^2.1.1:
```
Comment (attilah) [inline yarn.lock]: Please regen and review yarn.lock changes.

**Pull request merged**